src/http/http_client.cpp

```diff
@@ -52,13 +52,23 @@ HTTPClient::HTTPClient() {
     HTTPSessionInstantiator::registerInstantiator();
     HTTPSSessionInstantiator::registerInstantiator();
 
-    // Prepare for SSLManager
+#ifdef WIN32
+    // Windows implementation for the Context class
+    // is different to that on Linux.
+    Poco::SharedPtr<AcceptCertificateHandler> cert =
+        new AcceptCertificateHandler(false);
+    const Poco::Net::Context::Ptr context = new Context(
```

Comment:
Could we only keep this under `ifdef` and not the statement above and bellow? Those seem to be the same either way.